### PR TITLE
update dependencies and fix deprecated stuff

### PR DESCRIPTION
**Pull request by mamal72**

### CODE REVIEW

**Comment by mamal72** [comment]
@maxogden I wasn't sure to add myself to contributors or not. You can remove that line. That's OK to me. 👍 😁

**Comment by pangeacake** [comment]
squash into one version change?

**Comment by mamal72** [comment]
@pangeacake Sry I didn't get that right. Do you mean have them all as 4.2.0?

**Comment by pangeacake** [comment]
@mamal72 I was suggesting to maybe do that, yeah

**Comment by kingsloi** [comment]
`npm i --save-dev maxogden/menubar#pull/117/head` for all those who want this PR before it's merged!

File: package.json [hunk 1]
```diff
@@ -1,13 +1,14 @@
 {
   "name": "menubar",
-  "version": "4.1.0",
+  "version": "4.2.0",
```
Comment (jenslind) [inline package.json]: Don't bump the version, it's easier for us to do that when releasing to npm :)

File: package.json [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
   "description": "high level way to create menubar desktop applications with electron",
   "main": "index.js",
   "repository": {
     "type": "git",
     "url": "https://github.com/maxogden/menubar.git"
   },
   "scripts": {
+    "start": "electron .",
```
Comment (jenslind) [inline package.json]: Not sure why this was added?

File: package.json [hunk 3]
```diff
@@ -26,10 +27,10 @@
   },
   "homepage": "https://github.com/maxogden/menubar",
   "dependencies": {
-    "electron-positioner": "^2.0.1",
-    "extend": "^2.0.0"
+    "electron-positioner": "^2.0.3",
+    "extend": "^3.0.0"
```
Comment (jenslind) [inline package.json]: I see no point of updating the dependencies here.

**Comment by jenslind** [comment]
Thanks for the PR, added some inline comments.

Also think we should go with `opts.option` instead of `opts['option']` since other options that didn't contained `-` uses dots.

File: package.json (hunk 2 above)
Comment (mamal72) [inline package.json]: It's useful when user has `electron-prebuilt` or some electron bin package as the dependencies. I think it's better to omit it. 🤔

File: package.json (hunk 3 above)
Comment (mamal72) [inline package.json]: I think it's always good to have updated dependencies as far as they don't break your code. I see no point of don't updating. 😄 
Is it going to be a problem?

Comment (jenslind) [inline package.json]: `^2.0.1` will install `2.0.3` and for extend please see #60

Comment (mamal72) [inline package.json]: You are right. We can leave them as they are.

**Comment by mamal72** [comment]
@jenslind I think I should update all the options. Seems all of them are camelCased and I missed some of them. If you are OK with it, I'll push the changes on this PR.

**Comment by jenslind** [comment]
@mamal72 yes push any updates! :)

**Comment by jenslind** [comment]
Sorry, but closing in favor of #119 I want this fixed :)

**Pull request closed without merging**